samples/listFilesPaginate.js

```diff
@@ -14,6 +14,7 @@
 /* eslint-disable no-unused-vars */
 
 function main(bucketName = 'my-bucket') {
+  // @TODO There are two samples with ID: "storage_list_files"
```

Comment:
⚠️

Reply:
Great catch!! Can you please change the region tag for this sample to be `storage_list_files_paginated`?

Reply:
Done!